account_invoice_line_sequence/__manifest__.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2017 Camptocamp SA - Damien Crier, Alexandre Fayolle
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
+# Copyright 2017 Serpent Consulting Services Pvt. Ltd.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+{
+    'name': 'Account Invoice line with sequence number',
+    'version': '11.0.1.0.0',
+    "summary": "Adds sequence field on invoice lines to manage its order.",
+    'category': 'Accounting',
+    'author': "Camptocamp, "
+              "Eficent, "
+              "Serpent CS, "
+              "Odoo Community Association (OCA)",
+    'website': 'http://www.camptocamp.com',
```

Comment:
@amcor IMO Replace URL as per [this](https://agithub.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L10)